scripts/10-main.ldif.in

```diff
@@ -4,28 +4,27 @@
 dn: {{.BaseDN}}
 objectClass: top
 
-dn: ou={{.Company}},{{.BaseDN}}
-objectClass: organizationalUnit
+dn: o={{.Company}},{{.BaseDN}}
```

Comment:
Not totally sure about the organizationalUnit to organization change. The only real argument I have against it is however that you cannot make it a kopano-company if its not an organizationalunit.

https://github.com/Kopano-dev/kopano-core/blob/a16ffc825e4e7a6d6da81fa17cd52a59f41f3038/installer/ldap/kopano.schema#L247-L260

(but on the other hand the Kopano ldap schema definitions are not too relevant in the scope of this project)

Reply:
Mhm i see - i am already considering keeping the `ou` thing - but i also want to make it possible to differenciate between the different organizationalUnits.